### ISSUE TITLE
MozStumb... Apptitle is cut off on some devices

### ISSUE THREAD

**Issue by Djfe** [mozilla/MozStumbler#1052]
the app title is cut off on some devices which looks ugly IMO

workarounds:
we could move the leaderboard and settings buttons to the bottom of the drawer, make it scrollable and open on pressing the legacy menu key of the device

or we could move the on/off switch to the top-right off the drawer or make it narrower

the issue:
![2014-10-20 22 59 24](https://cloud.githubusercontent.com/assets/5685512/4710669/58221142-58ad-11e4-9619-3e66419f7597.png)

**Comment by garvankeeley** [comment]
Oh man! This 480px wide screen, the font is really large. What Android is this?

**Comment by Djfe** [comment]
Sony Xperia Arc S using
Carbonrom (basically Kitkat 4.4.4)

**Comment by Djfe** [comment]
I guess the padding of the on/off switch is too big
imagine: without the dots "ler" could definitely fit into that space
it would just be a tiny bit closer to the on/off switch than the buttons

**Comment by Djfe** [comment]
Works fine in landscape of course, but the font in landscape is smaller, too.

**Comment by Djfe** [comment]
I'm not even sure if this needs to be fixed:
pretty much all of these devices with such a small width have a legacy menu button, right?
so the three dots won't be there

I bet this issue will disappear from my device after an update (gonna test it again this weekend)

somehow my rom tells apps to show the ... menu button even though I have a hardware menu button

**Comment by Djfe** [comment]
Now it's Mozilla Stumbler...
that's even longer and too long for the app drawer

I thought MozStumbler is a catchy name, Mozilla Stumbler sounds too long

**Comment by garvankeeley** [comment]
Lots of apps have names about this length, just looking at my device: Amazon Kindle, ES File Explorer, News & Weather, Podcast Addict, Play Newsstand, Play Movies & TV. The name doesn't seem unusually long.

**Comment by Djfe** [comment]
yeah but the beginning of the name says something about the app unlike Mozilla which pretty much only says that it's from Mozilla

**Comment by gpoo** [comment]
Jsst to provide a little more information:

I think the issue with Android 4.4.4 which has that menu on the right (on a Nexus 5 looks ok).  With 4.0.4, which does not have the menu on the right, it looks ok:

![screenshot_2014-10-28-10-51-40](https://cloud.githubusercontent.com/assets/877181/4813920/b04ebb0c-5ecc-11e4-85f9-b7fff8e7ae97.png)

However, with 2.3.7; it only says "Mozilla S", but that before starting scanning (possibly because of the long text and the gear menu).  Here 3 consecutive screenshots, the first time running Mozilla Stumbler, after pressing "START SCANNING", and then after pressing the "pause" button.

![stumbler-099-android-237](https://cloud.githubusercontent.com/assets/877181/4813956/f6ae8ed8-5ecc-11e4-912f-b2f1a98f922a.png)

**Comment by gpoo** [comment]
I take my word back with respect to 2.3.7.  With the newest 0.102, there is no time to see "START SCANNING", because it starts automatically (something that previously was not happening with that particular Android).
